tensorflow_text/core/kernels/wordpiece_tokenizer.h

```diff
@@ -41,10 +41,11 @@ class WordpieceVocab {
 
 LookupStatus WordpieceTokenize(
```

Comment:
We have a few internal users using this C++ API directly, can you add an overloaded function (w/o max_bytes_per_subtoken) and calls this function (setting max_bytes_per_subtoken = max_bytes_per_token)?

Reply:
Sure, I have done that. I set the default to `max_bytes_per_word = 0` which has the same effect. 

I also made sure that the op is registered with a default value of 0 for the new attr. That should make this change backwards compatible with old serialized graphs.